htlcswitch/switch.go

```diff
@@ -2771,7 +2773,7 @@ func (s *Switch) BestHeight() uint32 {
 }
 
 // evaluateDustThreshold takes in a ChannelLink, HTLC amount, and a boolean to
-// determine whether the default dust threshold has been exceeded. This
+// determine whether the default fee threshold has been exceeded. This
```

Comment:
I'm going to remove this function so I don't think it makes sense to change. also nothing about this function has changed, it still computes only with the dust rather than commit fee + dust

Reply:
We changed the commentary from "dust threshold" to "fee threshold". The function is currently called "evaluateDustThreshold". I don't understand how this isn't a change to the semantics. Is this just checking if the dust *alone* blows the fee threshold?

Reply:
fixed